geometry_msgs/Pose simpleBodyPose
geometry_msgs/Pose simpleHeadPose

# joint configurations

# leg configuration
# hands pose

# velocity
# predicted waypoints
#string firstname
#string lastname

string id
bool locked
bool moving

# activity
